cmd/api/src/migrations/manifest.go

```diff
@@ -1,35 +1,68 @@
 // Copyright 2023 Specter Ops, Inc.
-// 
+//
 // Licensed under the Apache License, Version 2.0
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-// 
+//
 //     http://www.apache.org/licenses/LICENSE-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-// 
+//
 // SPDX-License-Identifier: Apache-2.0
 
 package migrations
 
 import (
 	"context"
 	"strings"
+	"time"
 
-	"github.com/specterops/bloodhound/src/version"
 	"github.com/specterops/bloodhound/dawgs/graph"
 	"github.com/specterops/bloodhound/dawgs/ops"
 	"github.com/specterops/bloodhound/dawgs/query"
 	"github.com/specterops/bloodhound/graphschema/ad"
 	"github.com/specterops/bloodhound/graphschema/azure"
 	"github.com/specterops/bloodhound/graphschema/common"
 	"github.com/specterops/bloodhound/log"
+	"github.com/specterops/bloodhound/src/version"
 )
 
+func Version_508_Migration(db graph.Database) error {
+	defer log.Measure(log.LevelInfo, "Migrating Azure Owns to Owner")()
+
+	return db.BatchOperation(context.Background(), func(batch graph.Batch) error {
+		return batch.Relationships().Filterf(func() graph.Criteria {
+			return query.And(
+				query.Kind(query.Start(), azure.Entity),
+				query.KindIn(query.End(), azure.ManagementGroup, azure.ResourceGroup, azure.Subscription, azure.KeyVault),
```

Comment:
I believe you'll also need to query for these end node kinds in the migration:
- `azure.AutomationAccount`
- `azure.ContainerRegistry`
- `azure.LogicApp`
- `azure.VM`
- `azure.VMScaleSet`
- `azure.WebApp`
- `azure.FunctionApp`
- `azure.ManagedCluster`

Reply:
Actually, it may not be necessary but I don't think it hurts anything to include these in the migration.